DataMigration/MainWindow.xaml.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Windows;
+using System.Configuration;
+using Npgsql;
+using DataMigration.PostgresDB;
+using DataMigration.DataLayer;
+using DataMigration.Logger;
+
+
+namespace DataMigration
+{
+    /// <summary>
+    /// Interaction logic for MainWindow.xaml
+    /// </summary>
+    public partial class MainWindow
+    {
+
+        private const int Limit =10;
+        private int _totalCount;
+
+        private Logger.Logger _logger = new Logger.Logger();
+        private readonly PostgresConnect _postgresConnect = new PostgresConnect();
+        private readonly SqlConnect _sqlConnect = new SqlConnect();
+
+
+        private List<HistoricalOcrData> _historicalOcrData = new List<HistoricalOcrData>();
+        private List<string> _updatedDocPaths = new List<string>();
+        private List<VanguardDoc> _vanguardDocsData = new List<VanguardDoc>();
```

Comment:
why do you use _historicalOcrData, _updatedDocPaths, _vanguardDocsData  as class fields instead of local variables?